### ISSUE TITLE
Conflicting, Duplicate PropertyType Listed Under Different Name in CFN Spec Files and Docs

### ISSUE THREAD

**Issue by ScriptAutomate** [awsdocs/aws-cloudformation-user-guide#502]
The [AWS::EC2::TrafficMirrorFilterRule.TrafficMirrorPortRangeRequest](https://github.com/awsdocs/aws-cloudformation-user-guide/blob/394cca0febdb73c34d6f62927b41f116e65e95c9/doc_source/aws-properties-ec2-trafficmirrorfilterrule-trafficmirrorportrangerequest.md) (only exists in GitHub repo docs, and most CFN spec files) looks like it is the same as [AWS::EC2::TrafficMirrorFilterRule.TrafficMirrorPortRange](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-trafficmirrorfilterrule-trafficmirrorportrange.html) (only exists on published Amazon CFN docs and some CFN spec files). Only one or the other are present in each CFN spec files.

This was discovered by an auditing tool in version `7.1.0` of the CFN spec files, at the bottom of the latest commit for: [ScriptAutomate/aws-cfn-resources-specs/supported-regions-per-resource.json](https://github.com/ScriptAutomate/aws-cfn-resource-specs/commit/c56677e62d8814cf3b500d9886942517f462823d#diff-aa248e60a2750348a511674a50b457ea) (due to missing as a `PropertyType` in `us-east-1`).

---

Regions where `AWS::EC2::TrafficMirrorFilterRule.TrafficMirrorPortRange` is listed:
- eu-central-1
- us-east-1
- eu-west-2

### AWS::EC2::TrafficMirrorFilterRule.TrafficMirrorPortRange

```json
"AWS::EC2::TrafficMirrorFilterRule.TrafficMirrorPortRange": {
  "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-trafficmirrorfilterrule-trafficmirrorportrange.html",
  "Properties": {
    "FromPort": {
      "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-trafficmirrorfilterrule-trafficmirrorportrange.html#cfn-ec2-trafficmirrorfilterrule-trafficmirrorportrange-fromport",
      "PrimitiveType": "Integer",
      "Required": true,
      "UpdateType": "Mutable"
    },
    "ToPort": {
      "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-trafficmirrorfilterrule-trafficmirrorportrange.html#cfn-ec2-trafficmirrorfilterrule-trafficmirrorportrange-toport",
      "PrimitiveType": "Integer",
      "Required": true,
      "UpdateType": "Mutable"
    }
  }
}
```

Regions where `AWS::EC2::TrafficMirrorFilterRule.TrafficMirrorPortRangeRequest` is listed, instead:
- ap-northeast-2
- us-gov-west-1
- eu-west-1
- ap-south-1
- sa-east-1
- eu-north-1
- ap-southeast-1
- ca-central-1
- us-east-2
- us-west-2
- us-gov-east-1
- us-west-1
- ap-northeast-1
- eu-west-3

### AWS::EC2::TrafficMirrorFilterRule.TrafficMirrorPortRangeRequest
```json
"AWS::EC2::TrafficMirrorFilterRule.TrafficMirrorPortRangeRequest": {
  "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-trafficmirrorfilterrule-trafficmirrorportrangerequest.html",
  "Properties": {
    "FromPort": {
      "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-trafficmirrorfilterrule-trafficmirrorportrangerequest.html#cfn-ec2-trafficmirrorfilterrule-trafficmirrorportrangerequest-fromport",
      "PrimitiveType": "Integer",
      "Required": true,
      "UpdateType": "Mutable"
    },
    "ToPort": {
      "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-trafficmirrorfilterrule-trafficmirrorportrangerequest.html#cfn-ec2-trafficmirrorfilterrule-trafficmirrorportrangerequest-toport",
      "PrimitiveType": "Integer",
      "Required": true,
      "UpdateType": "Mutable"
    }
  }
}
```

**Comment by ScriptAutomate** [comment]
This looks nearly to have been resolved after `v7.3.0` of the CFN specs.

The official PropertyType looks to have completely become `AWS::EC2::TrafficMirrorFilterRule.TrafficMirrorPortRange` across the board in:

- All CFN spec files
- AWS CFN User Guide: Live Documentation

It has yet to be reflected in the GitHub source repository, with still uses `AWS::EC2::TrafficMirrorFilterRule.TrafficMirrorPortRangeRequest` instead. This issue can be closed once the GitHub repository is updated to reflect the source repository being use by AWS on the CFN User Guide website documentation.

**Comment by ckesser** [comment]
This is now fixed in this repo. Thank you for tracking this, @ScriptAutomate!
